### ISSUE TITLE
Hi, whether you have compared with other open source codec model, HiFi-Codec?

### ISSUE THREAD

**Issue by yangdongchao** [descriptinc/descript-audio-codec#3]
https://github.com/yangdongchao/AcademiCodec

**Comment by yangdongchao** [comment]
> +1, dongchao would you apply your GRVQ to this repo and do an independent benchmark? It's hard to determine the actual performance of this model against competitors when the hyperparams are so vastly different.

Hi, I will try to use GRVQ to this repo.

**Comment by ritheshkumar95** [comment]
@fmac2000 we recently trained a 24 KHz model with DAC, with the same exact config as EnCodec.
See here for results: https://twitter.com/ritheshkumar_/status/1671886168106090501?s=20

TL;DR It's more than 2x better in quantitative metrics.

We don't plan (at the moment) to evaluate against AcademiCodec.  Since we're released all the training + evaluation code, we're open to community contribution for that comparison

**Comment by yangdongchao** [comment]
> 



> @yangdongchao @fmac2000 we recently trained a 24 KHz model with DAC, with the same exact config as EnCodec. See here for results: https://twitter.com/ritheshkumar_/status/1671886168106090501?s=20
> 
> TL;DR It's more than 2x better in quantitative metrics.
> 
> We don't plan (at the moment) to evaluate against AcademiCodec. Since we have released all the training + evaluation code, we're open to community contribution for that comparison

OK. It is a great work. I want to know whether you have any plan to release 16khz model? I believe it will be very helpful for audio generation tasks.

**Comment by wangtiance** [comment]
> @yangdongchao @fmac2000 we recently trained a 24 KHz model with DAC, with the same exact config as EnCodec. See here for results: https://twitter.com/ritheshkumar_/status/1671886168106090501?s=20
> 
> TL;DR It's more than 2x better in quantitative metrics.
> 
> We don't plan (at the moment) to evaluate against AcademiCodec. Since we have released all the training + evaluation code, we're open to community contribution for that comparison

Hi Rithesh, thanks for sharing your wonderful work! By the same exact config, do you mean the encoder/decoder architectures are the same (thus similar model size), so the improvement is attributed to RVQ method/loss function/activation function etc.?

**Comment by ritheshkumar95** [comment]
@wangtiance , exact config meant that it uses the same compression parameters (24 KHz input sampling rate, 24 kbps compression bitrate, 320 striding factor, 32 codebooks of 10-bit size each)

Model design (architecture, activation functions, loss functions) are different and a feature of the paper

**Comment by wangtiance** [comment]
@ritheshkumar95 
I see, thanks. Is there also a comparison in terms of computational complexity (model parameters, MAC ops etc)?

**Comment by ritheshkumar95** [comment]
No we have not done that

**Comment by listener17** [comment]
> > +1, dongchao would you apply your GRVQ to this repo and do an independent benchmark? It's hard to determine the actual performance of this model against competitors when the hyperparams are so vastly different.
> 
> 
> 
> Hi, I will try to use GRVQ to this repo.
Hi @yangdongchao: did you manage to train with this codebase?

**Comment by lifeiteng** [comment]
@wangtiance FYI
![10351688529319_ pic](https://github.com/descriptinc/descript-audio-codec/assets/3107839/02e875bb-c544-45f8-b540-c3a82023b550)

**Comment by wangtiance** [comment]
@lifeiteng Thanks for the summary. What's the HiFi Encodec? How's it different from encodec?

**Comment by lifeiteng** [comment]
> @lifeiteng Thanks for the summary. What's the HiFi Encodec? How's it different from encodec?

https://github.com/yangdongchao/AcademiCodec/tree/master/egs/Encodec_24k_240d

**Comment by fmac2000** [comment]
@yangdongchao hello dongchao did you get around to implementing GRVQ?
